cni/pkg/install/install.go

```diff
@@ -304,3 +311,51 @@ func setNotReady(isReady *atomic.Value) {
 	installReady.Record(0)
 	isReady.Store(false)
 }
+
```

Comment:
This LGTM. Once the next `libcni` release is cut and the container runtimes get it, we should get subdirectory-based loading "for free" and will no longer have conflicts anyway, and can probably nuke most of this watching/rewriting junk.